.github/actions/presubmit/index.js

```diff
@@ -0,0 +1,32 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+const LABEL = 'presubmit';
```

Comment:
Can you explain what this actually does? Just some comments would be nice—like I don't understand if the build passing adds the presubmit tag, or whether it needs to exist to trigger it, or ... ?